### PR TITLE
update doc migration

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

File: en/hosting_guide/migration.rst [hunk 1]
```diff
@@ -22,14 +22,14 @@ Here are the settings added to config.yml ;
...
 
-- collection_name: this value indicates what will be the name of the collection used to store the migration information
-- database_name: each time the name of the database is needed, this alias is used
-- dir_name: as the use of this bundles will result in generation of files, the target is specified
-- script_dir_name: same as dir_name, but this indicates where js script are located
-- name: arbitrary value used in output
-- namespace: this value should be consistent with dir_name
+- ``collection_name``: this value indicates what will be the name of the collection used to store the migration information
+- ``database_name``: each time the name of the database is needed, this alias is used
+- ``dir_name``: as the use of this bundles will result in generation of files, the target is specified
```
Comment (ngilain) [inline en/hosting_guide/migration.rst]: Which kind of files ? Are we talking about log files or dump files ? This is important to have a correct dir_name / location.

**Comment by ngilain** [comment]
I find this doc ambiguous for a hosting guide: the last part trully concerns a hoster, but the first parts seem to be more related to integrators. Or maybe they are not. The configuration is not clear about who should configure it and for what purpose.

**Comment by alavieille** [comment]
@itkg-ngilain pr updated

**Comment by ngilain** [comment]
:+1:

**Pull request merged**